platform-sdk/consensus-gossip-impl/src/main/java/com/hedera/service/gossip/impl/FallenBehindAction.java

```diff
@@ -14,7 +14,9 @@
  * limitations under the License.
  */
 
-package com.swirlds.platform.system.status.actions;
+package com.hedera.service.gossip.impl;
```

Comment:
Is this the best package name? I do not consider gossip a service, especially since we use the term service very specifically in the execution side of the consensus node. What is the overall plan for package naming as this refactor continues?